subsys/usb/class/cdc_acm.c

```diff
@@ -416,29 +418,38 @@ static void cdc_acm_do_cb(struct cdc_acm_dev_data_t *dev_data,
 		LOG_DBG("Device connected");
 		break;
 	case USB_DC_CONFIGURED:
-		cdc_acm_read_cb(cfg->endpoint[ACM_OUT_EP_IDX].ep_addr, 0,
-				dev_data);
+		LOG_INF("Device configured");
+		if (!dev_data->configured) {
+			LOG_ERR("create transfer on ep 0x%02x",
+				cfg->endpoint[ACM_OUT_EP_IDX].ep_addr);
+			cdc_acm_read_cb(cfg->endpoint[ACM_OUT_EP_IDX].ep_addr, 0,
+					dev_data);
+		}
 		dev_data->tx_ready = true;
 		dev_data->tx_irq_ena = true;
 		dev_data->rx_irq_ena = true;
 		dev_data->usb_status = status;
-		LOG_INF("Device configured");
+		dev_data->configured = true;
 		break;
 	case USB_DC_DISCONNECTED:
 		LOG_INF("Device disconnected");
 		cdc_acm_reset_port(dev_data);
 		dev_data->usb_status = status;
+		dev_data->configured = false;
```

Comment:
It seems that this should be done also on receiving the callback with `USB_DC_RESET` status, so perhaps `cdc_acm_reset_port()` would be a better place for clearing this flag?